src/Humanizer.Tests/InflectorTests.cs

```diff
@@ -21,45 +21,47 @@
 //IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 //CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-using System.Collections;
 using System.Collections.Generic;
+using Humanizer.Inflections;
 using Xunit;
 using Xunit.Extensions;
 
 namespace Humanizer.Tests
 {
-    public class InflectorTests 
+    public class InflectorTests
     {
         public readonly IList<object[]> PluralTestData = new List<object[]>();
 
+        // All singular/pluralize tests just need to demonstrate that they're 
+        // delegating to the default vocab.
         [Theory]
-        [ClassData(typeof(PluralTestSource))]
+        [InlineData("mouse", "mice")]
         public void Pluralize(string singular, string plural)
         {
-            Assert.Equal(plural, singular.Pluralize());
+            Assert.Equal(plural, Vocabularies.Default.Pluralize(singular));
```

Comment:
`Vocabularies.Default` now contains that big list of words, so the big list of words are tested against `Vocabularies.Default`.  `Inflector` now just delegates a call to `Vocabularies.Default`.  (If it were possible to verify that method call cleanly, I'd do that, but testing for equality here is pretty close.)

My original implementation allowed the user to pass a `Vocabulary` to `Pluralize()`, etc, (e.g. a dictionary in another language) but I took this out before submitting the PR as it was overkill for the immediate problem.

I'll be happy to get the tests however they need to be, but I just wanted to make sure my reasoning was clear.  Please let me know how to proceed.